File: Cohort1/Funmi/Chapter6/src/exercise630/GuessNumber.java
package exercise630;
import java.util.Scanner;
import java.security.SecureRandom;

public class GuessNumber {
	public static void getGuessNum() {
		SecureRandom random = new SecureRandom();
		Scanner input = new Scanner(System.in);
	
		int number ;
		
		
		int temp = 1 + random.nextInt(5);
		System.out.println("Enter the any number between 1 to 1000: ");
		
		do {
			
			number = input.nextInt();
//			guess++;
			if(number > temp) 
				System.out.println("Too high please retry other number:" +"  ");
			
			else if(number < temp) 
				System.out.println("Too low please try again: ");
			
				
			
		}while(number != temp);
		System.out.println("Congratulation! you have won my number by this: " + temp );
		
		input.close();
	
	}
	

	public static void main(String[] args) {
//	calling a static method.
		GuessNumber.getGuessNum();

	}

}

File: Cohort1/Akin/python/guessNumber.py
import random
 
number =  random.randint(0,100)

print("Hi-Lo Number Guessing Game: between 0 and 100 inclusive.")
print()

#User guess the number

user_guess = int(input("Guess the random number between 0 and 100"))

while 0 >= user_guess <= 100:
    if user_guess > number:
        print("Guess too high")
    elif user_guess < number:
        print("Guess too low")
    else:
        print("Guess correct!!!")
        break

    user_guess = int(input("Guess the random number between 0 and 100"))

else:
    print("You quit early, the number is", number)


